public/index.html

```diff
@@ -1,43 +1,18 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
```

Comment:
I think there is no need to clean it on every branch you work on. You gonna probably create a `diff` one time by doing so.